superduperdb/backends/ibis/query.py

```diff
@@ -716,7 +729,9 @@ def execute(self, db):
 
 
 def _compile_item(item, db, tables):
-    if hasattr(item, 'compile'):
+    if hasattr(item, 'compile') and isinstance(
+        getattr(item, 'compile'), types.MethodType
+    ):
```

Comment:
Some objects from ibis have rewritten the `__getattr...._` func and return a new object, then will all be true here.

Reply:
For example, If we sort in reverse order.
```python
table.select('this').order_by(ibis.desc('this')).limit(10).execute(db)
```